kura/org.eclipse.kura.core.configuration/src/main/java/org/eclipse/kura/core/configuration/ComponentConfigurationImpl.java

```diff
@@ -41,7 +42,7 @@ public String getPid() {
     }
 
     @Override
-    public Tocd getDefinition() {
+    public OCD getDefinition() {
```

Comment:
Major chance. org.eclipse.kura.core.configuration is not an API package but I think it is used as an API because there was no other way to create an instance of a ComponentConfiguration.
The package and the bundle versions are affected.